astropy/io/misc/asdf/tests/__init__.py

```diff
@@ -1,2 +1,17 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 # -*- coding: utf-8 -*-
+
+# Define a constant to know if the entry points are installed, since this impacts
+# whether we can run the tests.
+
+import pytest
+from pkg_resources import iter_entry_points
+
+entry_points = [entry.name for entry in iter_entry_points('asdf_extensions')]
+ASDF_ENTRY_INSTALLED = 'astropy' in entry_points and 'astropy-asdf' in entry_points
```

Comment:
Absolutely not necessary, but the following one-liner does work (I'm starting to like `set`):
```
ASDF_ENTRY_INSTALLED = {'astropy', 'astropy-asdf'} <= {entry.name for entry in iter_entry_points('asdf_extensions')}
```
saves you deleting `entry_points` afterwards...

Reply:
I also like set operations but in this case I find the original code a lot simpler to parse mentally, so I'd like to keep it as-is.